docs/dyn/compute_alpha.securityPolicies.html

```diff
@@ -177,9 +177,12 @@ <h3>Method Details</h3>
           "action": "A String", # The Action to preform when the client connection triggers the rule. Can currently be either "allow" or "deny()" where valid values for status are 403, 404, and 502.
           "preview": True or False, # If set to true, the specified action is not enforced.
           "match": { # Represents a match condition that incoming traffic is evaluated against. Exactly one of the fields must be set. # A match condition that incoming traffic is evaluated against. If it evaluates to true, the corresponding ?action? is enforced.
-            "srcIpRanges": [ # CIDR IP address range. Only IPv4 is supported.
-              "A String",
-            ],
+            "config": {
+              "srcIpRanges": [ # CIDR IP address range. Only IPv4 is supported.
+                "A String",
+              ],
+            },
+            'versionedExpr': 'SRC_IPS_V1'
```

Comment:
Please use double quotes for consistency (here and below).

I can confirm that the structure for this diff (and below) matches the current `securityPolicy` [resource spec](https://cloud.google.com/compute/docs/reference/rest/beta/securityPolicies) in the Cloud Armor API reference.